Localize/bump-versions.js

```diff
@@ -0,0 +1,114 @@
+const fs = require('fs');
+const path = require('path');
+const run = require('../ci/ci-util').run;
+const semver = require('semver');
+
+const currentSprint = parseInt(process.env['SPRINT']);
+
+function getChangedFilesList() {
+    return run('git --no-pager diff --name-only origin/master..Localization').split('\n');
+}
+
+function getTaskOrPackagePaths(paths, rootFolder, exclusions) {
+    let pathSet = new Set();
+    paths.filter(p => p.startsWith(rootFolder))
+        .map(p => p.slice(0, p.indexOf('/', rootFolder.length)))
+        .filter(p => fs.existsSync(path.resolve(p)) && fs.statSync(path.resolve(p)).isDirectory())
+        .forEach(p => pathSet.add(p));
+
+    for (let excludedTask of exclusions) {
+        pathSet.delete(`${rootFolder}${excludedTask}`);
+    }
+    return Array.from(pathSet);
+}
+
+function getTasksPaths(paths) {
```

Comment:
We don't have Tasks folder in this repo

Reply:
Yep - the idea here is to reuse the same scripts that we use in another repos for now, and rework it later as an internal common package that includes scripts for version bump, sending messages, etc.